Pipfile

```diff
@@ -0,0 +1,7 @@
+[packages]
+pandas = "*"
```

Comment:
`pandas` is unnecessary for basic csv handling, which can be done using the [standard library](https://docs.python.org/3/library/csv.html).

Reply:
The use of pandas was to leverage its dataframe handling of null values.

Reply:
Why would there be null values? I can't see any in the CSV file.

Reply:
They've only given us one csv file so far. However, its flexible so the business can provide and combination of lists with or without translations as they need to. While I don't see why they would want to do this the front-end translation implementation (based on a single list with multiple languages) allows for this and present only the list items present in the json for that language. If you don't want to present an item in a given language you they don't have to.

Reply:
nulls can be dropped as each row is read with:
```python
json_data = []
with open(source_file_location, newline='') as csvfile:
    for row in csv.DictReader(csvfile):
        json_data.append({k: v for k, v in row.items() if v is not None})
```

I think we should remove the use of pandas. This will remove the need for any package installation, so no Pipfile will be needed and the script can be simply run with `./scripts/convert_csv_to_json.py`.

Reply:
Done